.github/workflows/build.yaml

```diff
@@ -35,6 +35,8 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           push: true
-          tags: eventmesh/eventmesh:latest
           file: ./docker/Dockerfile
           context: ./
+          tags: |
+            eventmesh/eventmesh:latest
+            apache/eventmesh:latest
```

Comment:
I don't think adding `apache` to the script will help solve this issue. Actually this script never works correctly after commit https://github.com/apache/eventmesh/commit/2e6e6bfbdc3ee629eff93ef8804094c88f8bbc6b.

May you please make neccesary modifications and verify it works in your forked repo? Then we can set the DockerHub auth in repo ENVs.

Reply:
You want me to use my docker creds for my repo and than check if the image is pushed. Is my assumption correct?

Reply:
Yes, in this way, we will be able to locate the reason of function failure.

Reply:
Made the changes to the workflow. The docker image is getting pushed now. Please let me know if we need to update the repos where the image is to be pushed.